winui/Avatar-view/how-to.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: States in WinUI AvatarView control | Syncfusion
+description: Learn about using of the different states that exist in WinUi AvatarView control(SfAvatarView).
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# How to use different states
+
+## Set badge to avatar
+
+The [SfAvatarView](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.SfAvatarView.html) control provides support for [SfBadge](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Notifications.SfBadge.html) to notify users of new or unread messages, notifications, or the status of something.
+
```

Comment:
@MadhumithanJayapathi The SfAvatarView control supports various SfBadge states to notify users of new or unread messages, notifications, or status updates.

Reply:
@Manivannan-E Changed the content as mentioned.